src/Javis.jl

```diff
@@ -373,13 +366,14 @@ end
 
 export render, latex
 export Video, Object, BackgroundObject, Action, Rel
-export Line, Translation, Rotation, Transformation, Scaling
-export val, pos, ang, get_value, get_position, get_angle
+export Line, Transformation
+export val, pos, ang, scl, get_value, get_position, get_angle, get_scale
 export projection, morph_to
 export appear, disappear, rotate_around, follow_path, change
 export rev
 export scaleto
 export act!
+export anim_translate, anim_rotate, anim_rotate_around, anim_scale
```

Comment:
I am not a huge fan of the names `anim_rotate` and `anim_rotate_around`. I finally figured out what we could name these two things:

- `anim_rotate` stays `anim_rotate`
- `anim_rotate_around` could instead become `anim_revolve` where "revolve" is defined as rotation around a distant point.

Reply:
I like that `anim_rotate` will suggest `anim_rotate_around` in your editor. `revolve` would be much harder to find. One would probably need to do `?anim_rotate`. I personally don't find it that intuitive either (maybe because I'm not a native speaker)

Reply:
Hmmm... I agree with what you mean and the TAB suggestion is very convenient to have. I was borrowing the syntax idea of `revolve` from 3D modeling tools where that sort of syntax is commonly used. Let's keep the syntax the way it is.